config/locales/en.yml

```diff
@@ -256,6 +256,7 @@ en:
       chilly_bin: "Chilly Bin"
       search_results: "Search Results"
       epic: 'Epic'
+      history: 'History Of'
```

Comment:
I've kept `history` because it is the name of the column. `history_of` would make it confusing on a few situations like the column having a `history_of_column` class.